ipyparallel/client/asyncresult.py

```diff
@@ -362,7 +367,17 @@ def __getattr__(self, key):
         except (error.TimeoutError, KeyError):
             raise AttributeError("%r object has no attribute %r"%(
                     self.__class__.__name__, key))
-
+    
+    @staticmethod
+    def _wait_for_child(child, timeout=_FOREVER, evt=None):
+        """Wait for a child to be done"""
+        if evt is not None:
+            evt = Event()
+        else:
+            evt.unset()
+        child.add_done_callback(lambda f: evt.set())
```

Comment:
Hum... Nop. You mean `if evt is None`.. otherwise you call `evt.unset()` on `evt` which **is** `None` ...  and IIRC `None` has no attribute `unset`.

It can be that  I am tired.. or that you [forget your own tweets](https://twitter.com/minrk/status/720565923111968768).